pkg/ext-proc/scheduling/scheduler.go

```diff
@@ -107,13 +107,13 @@ type Scheduler struct {
 // PodMetricsProvider is an interface to provide set of pods in the backend and information such as
 // metrics.
 type PodMetricsProvider interface {
-	AllPodMetrics() []*backend.PodMetrics
+	AllFreshPodMetrics() []*backend.PodMetrics
 }
 
 // Schedule finds the target pod based on metrics and the requested lora adapter.
 func (s *Scheduler) Schedule(req *LLMRequest) (targetPod backend.Pod, err error) {
-	klog.V(logutil.VERBOSE).Infof("request: %v; metrics: %+v", req, s.podMetricsProvider.AllPodMetrics())
-	pods, err := s.filter.Filter(req, s.podMetricsProvider.AllPodMetrics())
+	klog.V(logutil.VERBOSE).Infof("request: %v; metrics: %+v", req, s.podMetricsProvider.AllFreshPodMetrics())
```

Comment:
Can you remove the `AllFreshPodMetrics()` from this logging? This was probably added in initial POC. This is really DEBUG level logging and can be very heavy if there are many pods in the pool

Reply:
@liu-cong 
Updated.